### ISSUE TITLE
[webview] ui bug

### ISSUE THREAD

**Issue by philos3** [flutter/flutter#125664]
### Is there an existing issue for this?

- [X] I have searched the [existing issues](https://github.com/flutter/flutter/issues)
- [X] I have read the [guide to filing a bug](https://flutter.dev/docs/resources/bug-reports)

### Steps to reproduce

use  webview_flutter: ^4.0.7  open the follow url ,and you will see some ui problems(ios & android ，some android open error), but it is normal when open outside APP in phone browser .

https://tronscan.org/#/transaction/bd51970b7322e83c91ea5552b49d515e7062c33795acc709e54aea115bbcde2a

![image](https://user-images.githubusercontent.com/8424317/235033399-f4d4121e-b9c2-4d80-b8ca-9dee6e191168.png)

### Expected results

fix  the ui problem

### Actual results

Some UI elements overlap

### Code sample

<details><summary>Code sample</summary>

```dart
String url ="https://tronscan.org/#/transaction/bd51970b7322e83c91ea5552b49d515e7062c33795acc709e54aea115bbcde2a"
WebViewController controller = WebViewController()..loadRequest(Uri.parse(url));

Navigator.push(context, MaterialPageRoute(
          builder: (context) {
            return WebViewWidget(
                  controller: controller,);
          },
        ));
```

</details>

### Screenshots or Video

<details>
<summary>Screenshots / Video demonstration</summary>

![image](https://user-images.githubusercontent.com/8424317/235034011-351c2380-98e7-430a-8828-85f6e3977f6a.png)

</details>

### Logs

<details><summary>Logs</summary>

```console
[Paste your logs here]
```

</details>

### Flutter Doctor output

<details><summary>Doctor output</summary>

```console
[✓] Flutter (Channel stable, 3.3.10, on macOS 12.5 21G72 darwin-arm, locale zh-Hans-CN)
    • Flutter version 3.3.10 on channel stable at /Users/linzechi/fvm/versions/3.3.10
    • Upstream repository https://github.com/flutter/flutter.git
    • Framework revision 135454af32 (4 months ago), 2022-12-15 07:36:55 -0800
    • Engine revision 3316dd8728
    • Dart version 2.18.6
    • DevTools version 2.15.0
    • Pub download mirror https://pub.flutter-io.cn
    • Flutter download mirror https://storage.flutter-io.cn

[✓] Android toolchain - develop for Android devices (Android SDK version 33.0.0)
    • Android SDK at /Users/linzechi/Library/Android/sdk
    • Platform android-33, build-tools 33.0.0
    • Java binary at: /Library/Java/JavaVirtualMachines/temurin-11.jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment Temurin-11.0.16.1+1 (build 11.0.16.1+1)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS (Xcode 14.0.1)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Build 14A400
    • CocoaPods version 1.12.0

[!] Android Studio (version 2022.1)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart
    ✗ Unable to find bundled Java version.
    • Try updating or re-installing Android Studio.


[✓] Connected device (2 available)
    • NAM AL00 (mobile) • EMHVB22324002820 • android-arm64 • Android 12 (API 31)
    • macOS (desktop)   • macos            • darwin-arm64  • macOS 12.5 21G72 darwin-arm

[✓] HTTP Host Availability
    • All required HTTP hosts are available

```

</details>

**Comment by philos3** [comment]
webview_flutter: ^4.0.7

**Comment by danagbemava-nc** [comment]
This is not an issue with the webview plugin. The site is just not properly optimized for safari mobile. You'll need to report it to whoever owns the site so that they can work on it. I opened the site in safari and I see the same view as you have shared above. 


<img src="https://user-images.githubusercontent.com/88313112/235075082-de6bb9ef-4d00-4db2-a8a0-8178eedd74c3.PNG" width="300"/>
<img src="https://user-images.githubusercontent.com/88313112/235075142-89f52444-8340-45cf-8042-31b701ab355e.PNG)" width="300"/>

Closing from here as this is not a flutter issue. 

Thank you
